interni/verbali/verbale-interno_2020-04-20.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass{article}
+
+\input{../../commons/config}
+
+\appendToGraphicspath{../../commons/img/}
+
+\title{Verbale interno --- 20/04/2020}
+
+\setResponsabile{Luca Ercole}
+\setRedattori{Alberto Cocco}
+\setVerificatori{
+  Alberto Gobbo
+}
+\setUso{Interno}
+\setDescrizione{Verbale dell'incontro di GruppOne del 20/04/2020}
+\setModifiche{%
+\cellcolor{white!80!lightgray!100} & Luca Ercole & 2020--04--21 & approva documento \\%
+\cellcolor{white!80!lightgray!100} & Alberto Gobbo & 2020--03--20 & verifica verbale \\%
+\multirow{-3}{*}{-} & Alberto Cocco & 2020--04--20 & stendi verbale%
+}
+
+\disabilitaVersione{}
+\disabilitaElencoFigure{}
+\disabilitaElencoTabelle{}
+
+\begin{document}
+
+\input{../../commons/template}
+
+\section{Informazioni logistiche}%
+\label{sec:informazioni_logistiche}
+
+\begin{description}
+  \item [Luogo] videochiamata su Google Hangouts
+  \item [Data] 20/04/2020
+  \item [Ora] 18:30 \symbol{8594} 20:00
+\end{description}
+
+\subsection{Membri del gruppo presenti}%
+\label{sub:membri_del_gruppo_presenti}
+
+\begin{enumerate}
+  \item Riccardo Agatea
+  \item Tobia Apolloni
+  \item Riccardo Cestaro
+  \item Alberto Cocco
+  \item Luca Ercole
+  \item Alberto Gobbo
+  \item Alessandro Rizzo
+  \item Fabio Scettro
+\end{enumerate}
+% sub:membri_del_gruppo_presenti (end)
+
+\section{Introduzione}%
+\label{sec:introduzione}
+
+L'incontro, che si è tenuto su Google Hangouts, è servito per ripianificare il lavoro.
+
+\section{Ordine del giorno}%
+\label{sec:ordine_del_giorno}
+\begin{itemize}
+  \item Aggiornamento foglio dei requisiti.
+  \item Manuale utente e manutentore.
+  \item Diagrammi UML.
+\end{itemize}
+
+\section{Aggiornamento foglio dei requisiti}%
+\label{sec:aggiornamento_foglio_dei_requisiti}
+
+Il gruppo ha controllato tutti i requisiti presenti nel foglio google creato in data 13/04 al fine di verificarne lo stato.
```

Comment:
"nel foglio Google creato in data 13/04/2020"

Reply:
Intendi Google con la maiuscola?

Reply:
Yes